rollup.config.js

```diff
@@ -7,12 +8,12 @@ export default [
     external: Object.keys(peerDependencies || {}),
     plugins: [
       typescript({
-        typescript: require('typescript'),
-      }),
+        typescript: require('typescript')
+      })
     ],
     output: [
       { file: main, format: 'cjs' },
-      { file: module, format: 'esm' },
-    ],
-  },
```

Comment:
Please, keep all trailing commas you removed.

Reply:
Eslint was complaining both when I had a trailing comma and without, which was why I changed it.